init.ts

```diff
@@ -42,17 +42,22 @@ const flags = parse(Deno.args, {
   default: { "force": null, "twind": null, "vscode": null },
 });
 
-if (flags._.length !== 1) {
-  error(help);
-}
-
 console.log(
   `\n%c  🍋 Fresh: the next-gen web framework.  %c\n`,
   "background-color: #86efac; color: black; font-weight: bold",
   "",
 );
 
-const unresolvedDirectory = Deno.args[0];
+let unresolvedDirectory = Deno.args[0];
+if (flags._.length !== 1) {
+  const userInput = prompt("Projet Name", "fresh-project");
```

Comment:
```suggestion
  const userInput = prompt("Project Name", "fresh-project");
```
... was probably the intention here.

Reply:
Good catch! Someone already made a PR #1269